templates/travis/.travis.yml.j2

```diff
@@ -143,6 +143,10 @@ jobs:
       before_script: skip
       script: bash .travis/cherrypick.sh
     {%- endif %}
+    - stage: test-released-plugin-with-pulpcore-master-branch
+      env:
+        - TEST=plugin-from-pypi
+      if: type = cron
```

Comment:
what happens when we have a breaking change?
Taking drf-yasg to drf-spectacular example, I believe we would break the majority of the CIs with this.
I think we need some conditional for enabling this job

Reply:
@fao89 The goal is to not have breaking changes like that. That's why this check should always be there so plugins can notify pulpcore devs that they made a change that was backwards incompatible.